pkg/api/project.go

```diff
@@ -23,11 +25,34 @@ func GetProject(client graphql.Client, orgID string, idOrSlug string) (*Project,
 func (p *getProjectByIdProject) toProject() *Project {
 	return &Project{
 		ID:            p.Id,
+		Name:          p.Name,
 		Slug:          p.Slug,
+		Description:   p.Description,
 		DefaultParams: p.DefaultParams,
 	}
 }
 
+func (p *projectsProjectsProject) toProject() Project {
+	return Project{
+		ID:            p.Id,
+		Slug:          p.Slug,
+		Name:          p.Name,
+		Description:   p.Description,
+		DefaultParams: p.DefaultParams,
+	}
+}
+
+func ListProjects(client graphql.Client, orgID string) (*[]Project, error) {
```

Comment:
```suggestion
func ListProjects(client graphql.Client, orgID string) ([]Project, error) {
```

Slices are already pointers, so `*[]...` is 2 levels of indirection (a pointer to a pointer to an array) and thus unnecessary.